RUFAS/routines/field/soil/layer_data.py

```diff
@@ -424,12 +426,17 @@ def _initialize_carbon_pools(self, field_size: float) -> None:
         soil_volume_in_cubic_meters = self.layer_thickness * (field_size * HECTARES_TO_SQUARE_MILLIMETERS) * \
             CUBIC_MILLIMETERS_TO_CUBIC_METERS
         soil_mass_in_kg = self.bulk_density * MEGAGRAMS_TO_KILOGRAMS * soil_volume_in_cubic_meters
+        total_soil_carbon_amount = (soil_mass_in_kg * (self.percent_organic_carbon_content / 100) / field_size)
 
-        one_third_soil_carbon = (1 / 3) * (soil_mass_in_kg * (self.percent_organic_carbon_content / 100) / field_size)
-
-        self.active_carbon_amount = one_third_soil_carbon
-        self.passive_carbon_amount = one_third_soil_carbon
-        self.slow_carbon_amount = one_third_soil_carbon
+        if self.top_depth == 0:
+            self.active_carbon_amount = 0.02 * total_soil_carbon_amount
+            self.slow_carbon_amount = 0.98 * total_soil_carbon_amount
+            self.structural_litter_amount = (1 / 2) * residue
+            self.metabolic_litter_amount = (1 / 2) * residue
+        else:
+            self.active_carbon_amount = 0.02 * total_soil_carbon_amount
+            self.slow_carbon_amount = 0.54 * total_soil_carbon_amount
+            self.passive_carbon_amount = 0.44 * total_soil_carbon_amount
 
```

Comment:
```suggestion
            self.passive_carbon_amount = 0.44 * total_soil_carbon_amount
            self.structural_litter_amount = 0.0
            self.metabolic_litter_amount = 0.0
```
We want to really be explicit in making sure there is no belowground litter (for now) because some layers get post_init called on them once when they get initialized, then get modified, then get post_init called on them again.